Bottom: a55beef151ff1afe46ddb45fb4beaae38f67b6d8
Top:    623a475191efa43ebea39d2b09a61e581a20047d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-02 18:32:34 +0100

Fix array index out of bounds over layer groups



---

diff --git a/src/move.c b/src/move.c
index 2e9eb1f..003c2e0 100644
--- a/src/move.c
+++ b/src/move.c
@@ -1027,7 +1027,7 @@ MoveLayer (int old_index, int new_index)
 
   move_all_thermals(old_index, new_index);
 
-  for (g = 0; g < MAX_LAYER + 1; g++)
+  for (g = 0; g < MAX_LAYER; g++)
     PCB->LayerGroups.Number[g] = 0;
   for (l = 0; l < max_layer + 2; l++)
     {
